File: A2/q2.py
def get_value_from_array(array, index):
    # Retrieve the element from the array at the specified index
    return array[index]

def get_value_from_array2(array, index):
    if index < 0 or index >= len(array):
        # Handle invalid index (e.g., return None or raise an exception)
        return None  # Or use: raise IndexError("Index out of range")
    return array[index]

File: A2/q2.c

int getValueFromArray(int *array, int len, int index) {
    // Retrieve the element from the array at the specified index
    return array[index];
}


int getValueFromArray2(int *array, int len, int index) {
    if (index < 0 || index >= len) {
        // Handle invalid index (e.g., return an error code)
        return -1; // Alternatively, you could print an error message or handle differently
    }
    return array[index];
}

